src/styles/blocks/_container.scss

```
@@ -0,0 +1,10 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 0;
+  width: wrapper(
+    $card-width,
+    $card-margin-right,
+    $line-card);
+  margin: 0 auto;
```

You shouldn't use external styles on blocks

This conrainer have display flex, so you can just use this for centering
```suggestion
  justify-content: center;
```